Backup/Simple_Backup_Service_FAQs.md

```diff
@@ -0,0 +1,140 @@
+{{{
+  "title": "Simple backup FAQs",
+  "date": "02-2-2016",
+  "author": "Justin Withington, John Gerger",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+**FAQs: Simple Backup Service**
+-------------------------------
+
+#### IMPORTANT NOTE
+
+CenturyLink Cloud’s Simple Backup Service product is currently in a Limited Beta with specific customers by invitation only and is not intended for production usage. During the Limited Beta there is no production Service Level Agreement.
+
+**FREQUENTLY ASKED QUESTIONS**
+------------------------------
+
+**Q: Where do I configure backups?**
+
+A: The Control Portal provides backup configuration functionality. Please refer to our [Getting Started Guide](./getting-started-with-simple-backup.md) for more details
+
+**Q: What are the network requirements for SBS, if any?**
+
+A: Simple Backup Service requires outbound internet traffic over port 443. CLC VMs allow outbound traffic by default using NAT.
+
+**Q: If the backup frequency is every 4 hours, what should I expect?**
+
+A: This depends on how long a backup takes to complete. The frequency timer will not start until the previous backup has completed, so in this example the next backup will start 4 hours after the previous one has completed.
+
+**Q: Can I manually initiate a backup outside of my regularly scheduled frequency?**
+
+A: Yes, from the Backup Agent, users can select the “Backup” button from the Home Dashboard or the Policy Details page. This will place a request at the top of the backup queue and will be processed as soon possible.
+
+**Q: Can I schedule backups to execute at a specific time in the day?**
+
+A: No, the policy frequency determines when the next backup will be executed or the user may manually trigger a backup. Once a backup has completed, the frequency will start to countdown and the next backup will occur when the frequency countdown has expired.
+
+**Q: Can I completely delete my backed up files from storage regardless of the retention period?**
+
+A: Not manually at this time.
```

Comment:
the Answer here sounds off.  did you mean No, this is a manual process at this time?